internal/storage/v2/elasticsearch/tracestore/fixtures/es_01.json

```diff
@@ -42,7 +42,7 @@
     {
       "key": "peer.ipv4",
       "type": "int64",
-      "value": "23456"
```

Comment:
ok so one issue here is that you're making this change in v2 implementation which is not going through any integration tests yet. Could we try making similar change in v1, just to see that the CI stays green?